legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/dbTestRunner/shared.pure

```diff
@@ -0,0 +1,87 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import meta::relational::runtime::*;
+import meta::relational::dbTestRunner::*;
+import meta::pure::alloy::connections::*;
+
+Profile meta::relational::dbTestRunner::dbTest
+{
+   stereotypes : [ Test, Ignore ];
+}
+
+Class meta::relational::dbTestRunner::DbTestConfig
+{
+   dbType: DatabaseType[1];
+   connection: RelationalDatabaseConnection[0..1];
+   expectedSql: String[0..1];
+}
+
+function meta::relational::dbTestRunner::collectTests(pack: Package[1]): Function<{DbTestConfig[1] -> String[0..1]}>[*]
+{
+   $pack.children->map(c| $c->match([
+      f:Function<Any>[1]| if($f.stereotypes->contains(dbTest->stereotype('Test')), |$f->cast(@Function<{DbTestConfig[1] -> String[0..1]}>), |[]),
+      p:Package[1]| collectTests($p),
+      x:Any[1]| []
+   ]));
+}
+
+function meta::relational::dbTestRunner::runTestsWith(
+                                               allTests: Function<{DbTestConfig[1] -> String[0..1]}>[*],
+                                               ignoredTests : Function<{DbTestConfig[1] -> String[0..1]}>[*],
```

Comment:
we don't need ignoredtests here, we can write the ignoring related logic in the calling function itself, and just pass valid tests in allTests

Reply:
passed ignoredTests to print summary here , else summary print needed to be written over n over

Reply:
ok. makes sense